### PR TITLE
Add Jdbi2Builders and accompanying tests

### LINKED ISSUE #165: Add Jdbi2Builders

### PR DESCRIPTION
**Pull request by chrisrohr**
Fixes #165

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
         <commons-io.version>2.6</commons-io.version>
         <commons-text.version>1.8</commons-text.version>
         <dropwizard.version>2.0.8</dropwizard.version>
+        <dropwizard.jdbi.version>2.0.7</dropwizard.jdbi.version>
```
Comment (sleberknight) [inline pom.xml:22]: Let's change this to `dropwizard.modules.dropwizard-jdbi` to attempt to make it more clear why it has a separate version. Does that make sense?

File: pom.xml [hunk 2]
```diff
@@ -196,6 +197,44 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```
Comment (sleberknight) [inline pom.xml:200]: Do we want to add another comment here mentioned that we have to exclude the dropwizard deps b/c the `dropwizard-jdbi` moved to modules and is not the same version?

File: pom.xml (hunk 1 above)
Comment (chrisrohr) [reply pom.xml:22]: Sure

File: pom.xml (hunk 2 above)
Comment (chrisrohr) [reply pom.xml:200]: Do you want the comment above the dep or above the exclusions?

Comment (sleberknight) [reply pom.xml:200]: I don't care...wherever it looks better? :-)

**Pull request merged**